ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,503 @@
+                button.translatesAutoresizingMaskIntoConstraints = false
+                self.button = button
+                addSubview(button)
+            }
+        }
+
+        setupLayoutConstraints()
+    }
+
+    open override func layoutSubviews() {
+        super.layoutSubviews()
+
+        // Round the top corners to fit the image inside the card
+        if let imageView = imageView {
+            let roundTopCornersMask = CAShapeLayer()
+            roundTopCornersMask.path = UIBezierPath(roundedRect: imageView.frame, byRoundingCorners: [.topLeft, .topRight], cornerRadii: CGSize(width: Constants.borderRadius, height: Constants.borderRadius)).cgPath
+            imageView.layer.mask = roundTopCornersMask
+            imageView.layer.masksToBounds = true
+        }
+    }
+
+    /// Set up the background color of the card and update the icon and text color if necessary
+    private func setupBackgroundColor() {
+        var color: UIColor = Constants.defualtBackgroundColor
+
+        switch backgroundStyle {
+        case .appColor:
+            if let window = window {
+                color = Colors.primaryTint40(for: window)
+                primaryLabel.textColor = Colors.primaryShade20(for: window)
+                secondaryLabel.textColor = Colors.primaryShade20(for: window)
+                iconView?.tintColor = Colors.primaryShade20(for: window)
+            }
+        case .neutral:
+            color = Colors.Button.background // TODO: will be updated to the right value once I have the values from design
+        case .custom:
+            color = cardBackgroundColor
+        }
+
+        self.backgroundColor = color
+    }
+
+    private struct Constants {
+        static let defualtBackgroundColor: UIColor = Colors.surfacePrimary
+        static let borderWidth: CGFloat = UIScreen.main.devicePixel
+        static let borderRadius: CGFloat = 8.0
+        static let borderColor = UIColor(light: Colors.gray100, dark: .clear)
+        static let horizontalCardPaddingTopBottom: CGFloat = 6.0
+        static let verticalCardPaddingBottom: CGFloat = 8.0
+        static let verticalCardPaddingTop: CGFloat = 10.0
+        static let cardPaddingTrailing: CGFloat = 16.0
+        static let xSmallPaddingLeading: CGFloat = 12.0
+        static let cardPaddingLeading: CGFloat = 10.0
+        static let horizontalContentSpacing: CGFloat = 12.0
+        static let verticalContentSpacing: CGFloat = 5.0
+        static let titleDefaultNumberOfLines: Int = 2
+        static let subtitleDefaultNumberOfLines: Int = 2
+        static let smallContentSpacing: CGFloat = 2.0
+        static let largeContentSpacing: CGFloat = 6.0
+        static let largeHorizontalPadding: CGFloat = 16.0
+        static let largeVerticalPadding: CGFloat = 14.0
+    }
+
+    private var iconView: UIImageView?
+
+    private var button: UIButton?
+
+    private var imageView: UIImageView?
+
+    private let primaryLabel: UILabel = {
```

Comment:
Is primaryLabel always going to be used? If not, I would suggest making it a lazy variable so that it only gets created if it's being used. Same for secondaryLabel.

Reply:
Yeah, all cards have a title